src/test/java/dev/farhan/movieist/MovieistApplication_corsConfigurer_7322b2b140_Test.java

```diff
@@ -0,0 +1,35 @@
+package dev.farhan.movieist;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MovieistApplication_corsConfigurer_7322b2b140_Test {
+
+    @Test
+    public void testCorsConfigurer() {
+        MovieistApplication movieistApplication = new MovieistApplication();
+        WebMvcConfigurer webMvcConfigurer = movieistApplication.corsConfigurer();
+
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        request.setMethod("GET");
+        request.addHeader("Origin", "http://example.com");
+
+        MockHttpServletResponse response = new MockHttpServletResponse();
+
+        webMvcConfigurer.getCorsConfiguration(request, response);
+
+        CorsConfiguration corsConfiguration = response.getCorsConfiguration();
```

Comment:
Error:The method getCorsConfiguration(MockHttpServletRequest, MockHttpServletResponse) is undefined for the type WebMvcConfigurer